from django.core.exceptions import ValidationError
from django.test import TestCase

from quizes.models import Quiz
